docs/_static/api.json

```diff
@@ -2343,8 +2364,9 @@
         },
         "/pulp/api/v3/content/rpm/packagecategories/": {
             "get": {
-                "operationId": "pulp_api_v3_content_rpm_packagecategories_list",
+                "operationId": "content_rpm_packagecategories_list",
```

Comment:
you need to rebase, it is bringing other plugins

Reply:
some docstrings aren't right:
https://github.com/pulp/pulp_ansible/compare/master...fao89:api_json